Tests/OgreMain/include/RootWithoutRenderSystemFixture.h

```diff
@@ -36,9 +36,9 @@ THE SOFTWARE.
 
 class RootWithoutRenderSystemFixture : public ::testing::Test {
 public:
-    Ogre::Root* mRoot;
-    Ogre::HardwareBufferManager* mHBM;
-    Ogre::FileSystemLayer* mFSLayer;
+    std::shared_ptr<Ogre::Root> mRoot;
+    std::unique_ptr<Ogre::HardwareBufferManager> mHBM;
```

Comment:
you can fix the error by instead moving mHBM on top of mRoot

Reply:
fixed